core/unit_test/TestViewCtorDimMatch.hpp

```diff
@@ -131,7 +131,7 @@ struct StaticRank<0> {
 TEST(TEST_CATEGORY_DEATH, view_construction_with_wrong_params_stat) {
   ::testing::FLAGS_gtest_death_test_style = "threadsafe";
 
-#ifndef KOKKOS_ENABLE_DEBUG_BOUNDS_CHECKS
+#ifndef KOKKOS_ENABLE_DEBUG_BOUNDS_CHECK
```

Comment:
so did the test pass? Or did we just not test with bounds check?

Reply:
The `'CUDA-11.6-NVCC-DEBUG` build has `Kokkos_ENABLE_DEBUG_BOUNDS_CHECK=ON`.